policies/templates/gcp_resource_value_pattern_v1.yaml

```diff
@@ -21,38 +21,54 @@ spec:
     spec:
       names:
         kind: GCPResourceValuePatternConstraintV1
-        plural: gcpresourcevaluepatternconstraintsv1
       validation:
         openAPIV3Schema:
           properties:
+            mode:
+              type: string
+              enum: [allowlist, denylist]
+              description: "String identifying the operational mode, allowlist or denylist.
+              In allowlist mode, resources with field_name values that do NOT
+              match the pattern defined will generate a violation.
+              In denylist mode, resources with field_name values that match the
+              pattern defined will generate a violation.
+              Defaults to allowlist."
```

Comment:
nit: this could go in a separate property (default: "allowlist")

Reply:
Ack

Reply:
sorry to be nit picky but I believe you need to include " " around that value since the type is string. Not sure if that would pass a YAML linter as-is

Reply:
Ack